code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -373,6 +381,7 @@
 	process_flags = ORGANIC | SYNTHETIC		//Admin-bus has no brakes! KILL THEM ALL.
 	metabolization_rate = 1
 	can_synth = 0
+	taste_message = null
```

Comment:
"admin abuse" :^)
Unless you don't want anyone knowing they are being adminbussed